Update configuration-db.md (#1691)
ManageIQ/manageiq-documentation: _includes/configuration-db.md

> <div class="important">

Reviewer: @meghasalunke1 was removing this intentional?

Author: Yes, Adam because in v3.5.0 documentation, this topic is not appearing. I checked other topics to understand what's wrong, then noticed these extra tags were causing the problem. Removing this should fix the issue.

        All {{ site.data.product.title_short }} appliances in a multi-region
        deployment must use the same key.

6.  Choose **Create Internal Database** for the database location. 

>     In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed. As a resolution, ensure to add the database disk instructions as defined above.

Reviewer: Sorry I still think this is confusing.  `when you select **Make No messaging changes**, then the FAILURE message is displayed` insinuates that this is going to happen every time.

Prefer something like:
```suggestion
    In the **Configure Messaging** menu select **Make No messaging changes**.  If you see ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` displayed make sure to add a second disk for the database per instructions defined above.
```

Author: Updated as suggested, please check and confirm. TIA

        All {{ site.data.product.title_short }} appliances in a multi-region
        deployment must use the same key.

6.  Choose **Create Internal Database** for the database location. 

>     In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed.

Author: @agrare Would you please reshare the [above](https://github.com/ManageIQ/manageiq-documentation/pull/1691/files#diff-e6503c312761f591c879d6d8313cb6a4db6bf696803dd65d500e104b4d59635aR5) link because when I click, it's taking me to the same comment that you mentioned.

>     In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed.

Reviewer: @meghasalunke1 it points to line 5 on this file: `Before installing an internal database, add a disk to the infrastructure
hosting your appliance.`

>     In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed.

Reviewer: @meghasalunke1 could you please make the change addressing this?

>     In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed.

Author: Added changes, @Fryguy I couldn't give the reference to the same page, would you please help, instead of adding 'above', I want to provide link for 'Configuring an Internal Database' (Title).
